docs/edgeql/select.rst

```diff
@@ -0,0 +1,671 @@
+``Villain`` type.
+
+.. code-block:: edgeql-repl
+
+  db> select Villain {name}
+  ... filter .name = "Doc Ock"
+  {default::Villain {name: 'Doc Ock'}}
+
+Filtering by ID
+^^^^^^^^^^^^^^^
+
+To filter by ``id``, remember to cast the desired ID to :ref:`uuid
+<ref_std_uuid>`:
+
+.. code-block:: edgeql-repl
+
+  db> select Villain {id, name}
+  ... filter .id = <uuid>"b233ca98-3c23-11ec-b81f-6ba8c4f0084e"
+  {
+    default::Villain {
+      id: 'b233ca98-3c23-11ec-b81f-6ba8c4f0084e',
+      name: 'Doc Ock'
+    }
+  }
+
+Nested filters
+^^^^^^^^^^^^^^
+
+Filters can be added at every level of shape nesting. The query below applies a
+filter to both the selected ``Hero`` objects and their linked ``villains``.
+
+.. code-block:: edgeql-repl
+
+  db> select Hero {
+  ...   name,
+  ...   villains: {
+  ...     name
+  ...   } filter .name ilike "%er"
+  ... } filter .name ilike "%man";
+  {
+    default::Hero {
+      name: 'Iron Man',
+      villains: {default::Villain {name: 'Justin Hammer'}},
+    },
+    default::Hero {
+      name: 'Spider-Man',
+      villains: {
+        default::Villain {name: 'Shocker'},
+        default::Villain {name: 'Tinkerer'},
+        default::Villain {name: 'Kraven the Hunter'},
+      },
+    },
+  }
+
+Note that the *scope* changes inside nested shapes. When we use ``.name`` in
+the outer ``filter``, it refers to the name of the hero. But when we use ``.
+name`` in the nested ``villains`` shape, the scope has changed to ``Villain``.
+
+.. _ref_eql_select_order:
+
+Ordering
+--------
+
+Order the result of a query with an ``order by`` clause.
+
+.. code-block:: edgeql-repl
+
+  db> select Villain { name }
+  ... order by .name
+  {
+    default::Villain {name: 'Abomination'},
+    default::Villain {name: 'Doc Ock'},
+    default::Villain {name: 'Green Goblin'},
+    default::Villain {name: 'Justin Hammer'},
+    default::Villain {name: 'Kraven the Hunter'},
+    default::Villain {name: 'Loki'},
+    default::Villain {name: 'Shocker'},
+    default::Villain {name: 'The Vulture'},
+    default::Villain {name: 'Tinkerer'},
+    default::Villain {name: 'Zemo'},
+  }
+
+The expression provided to ``order by`` can be any primitive singleton
+expression, including arrays and tuples.
+
+.. note::
+
+  When ordering by arrays or tuples, the leftmost elements are compared. If
+  these elements are the same, the next element is used to "break the tie", and
+  so on. If all elements are the same, the order is not well defined.
+
+.. code-block:: edgeql-repl
+
+  db> select Villain { name }
+  ... order by .name desc
+  {
+    default::Villain {name: 'Zemo'},
+    ...
+    default::Villain {name: 'Abomination'}
+  }
+
+For a full reference on ordering, including how empty values are handles, see
+:ref:`Reference > Commands > Select <ref_reference_select_order>`.
+
+
+.. _ref_eql_select_pagination:
+
+Pagination
+----------
+
+EdgeDB supports standard ``limit`` and ``offset`` operators. These are
```

Comment:
`limit` and `offset` are customary, but certainly not *standard* in any meaningful way, so I'd just omit the word "standard" here.